src/nanoserver/1809/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1809
+
+SHELL ["cmd", "/S", "/C"]
+USER ContainerAdministrator
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+RUN curl -SL --output %TEMP%\python.zip https://www.nuget.org/api/v2/package/python/3.7.3 \
+    && md C:\Python C:\PythonTemp \
+    && tar -zxf %TEMP%\python.zip -C C:\PythonTemp \
+    && xcopy /s c:\PythonTemp\tools C:\Python \
+    && rd /s /q c:\PythonTemp \
+    && del /q %TEMP%\python.zip \
+    && setx /M PYTHONPATH "C:\Python\Lib;C:\Python\DLLs;"
+
+ENV PATH="$WindowsPATH;C:\Program Files\PowerShell\;C:\Python;C:\python\scripts"
```

Comment:
Why using $WindowsPATH versus $PATH here?

Reply:
No specific reason, it's what the guys who made the image suggested and it worked.